src/gui/editorwidgets/qgsvaluerelationwidgetwrapper.cpp

```diff
@@ -365,6 +365,7 @@ void QgsValueRelationWidgetWrapper::populate( )
     for ( const QgsValueRelationFieldFormatter::ValueRelationItem &element : qgis::as_const( mCache ) )
     {
       whileBlocking( mComboBox )->addItem( element.value, element.key );
+      mComboBox->setItemData( mComboBox->count() - 1, element.description, Qt::ToolTipRole );
```

Comment:
`element.description` might be an empty string. Better go with an invalid `QVariant` for the cases where no expression were given.

Reply:
```suggestion
      if ( !element.description.isEmpty() )
        mComboBox->setItemData( mComboBox->count() - 1, element.description, Qt::ToolTipRole );
```